File: TestVectorsAwsCryptographicMaterialProviders/runtimes/net/tests/Extern/GetTestDirectoryExtern.cs
using byteseq = Dafny.Sequence<byte>;

namespace TestWrappedMaterialProvidersMain_Compile {
  public partial class __default {
    public static Dafny.ISequence<char> GetTestVectorExecutionDirectory() {
        return byteseq.FromString("");
    }
  }
}

File: TestVectorsAwsCryptographicMaterialProviders/runtimes/python/test/internaldafny/extern/GetTestDirectoryExtern.py
import TestWrappedMaterialProvidersMain
import os
import _dafny

def GetTestVectorExecutionDirectory():
    return _dafny.Seq(os.getcwd() + "/../../")

TestWrappedMaterialProvidersMain.default__.GetTestVectorExecutionDirectory = GetTestVectorExecutionDirectory